app/avo/actions/release_reserved_namespace.rb

```diff
@@ -0,0 +1,18 @@
+class ReleaseReservedNamespace < BaseAction
+  self.name = "Release reserved namespace"
+  self.visible = lambda {
+    current_user.team_member?("rubygems-org") && view == :show
+  }
+
+  self.message = lambda {
+    "Are you sure you would like to Release #{record.name} namespace?"
+  }
+
+  self.confirm_button_label = "Release namespace"
+
+  class ActionHandler < ActionHandler
+    def handle_model(rubygem)
+      rubygem.update_attribute(:updated_at, 101.days.ago)
```

Comment:
Seems this is not easily callable from console or any other code. What about to hide this logic into model `rubygem.release_reserved_namespace!` and just call it in here? Same change can be applied to https://github.com/rubygems/rubygems.org/blob/master/script/release_reserved_namespace#L11 (which is handy to be kept around still).

Reply:
The goal is that we only perform these actions through the admin dashboard and not via the scripts, since the scripts aren't audited

Reply:
That's clear, but I would prefer in case of emergency still be able to to simply call this method in console. There is no reason to mix app logic into avo actions. We can indeed delete scripts once we can fully rely on dashboard.

Reply:
I agree that we can move this into a method on the model. Done here 6efc8c97be7da046eced3fe40068d5499d86da93